cse-runner-app/src/main/java/com/farao_community/farao/cse/runner/app/services/FileExporter.java

```diff
@@ -104,10 +110,22 @@ String saveTtcResult(Timestamp timestamp) throws IOException {
         } catch (JAXBException e) {
             throw new CseInternalException("XSD matching error");
         }
-
         byte[] bytes = bos.toByteArray();
         InputStream is = new ByteArrayInputStream(bytes);
-        minioAdapter.uploadFile("outputs/ttc_res.xml", is);
-        return minioAdapter.generatePreSignedUrl("outputs/ttc_res.xml");
+        minioAdapter.uploadFile("outputs/" + getFilename(processTargetDate, processType), is);
+        return minioAdapter.generatePreSignedUrl("outputs/" + getFilename(processTargetDate, processType));
+    }
+
+    private String getFilename(OffsetDateTime processTargetDate, ProcessType processType) {
+        String filename;
+        if (processType == ProcessType.D2CC) {
+            ZonedDateTime targetDateInEuropeZone = processTargetDate.atZoneSameInstant(ZoneId.of("Europe/Paris"));
+            String dateAndTime = DateTimeFormatter.ofPattern("yyyyMMdd_HHmm").withLocale(Locale.FRANCE).format(targetDateInEuropeZone);
```

Comment:
you can do `targetDateInEuropeZone.format(DateTimeFormatter.ofPattern("yyyyMMdd_HHmm"))`